bin/use_simulator.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+
+set -o pipefail
+
+project="$1"
+scheme="$2"
+
+xcodebuild -project "$project" -scheme "$scheme" -showBuildSettings \
+  | grep CORRESPONDING_SIMULATOR_SDK_NAME \
+  2>&1 > /dev/null
```

Comment:
For one, `2>&1 >/dev/null` can be `&>/dev/null`.

I think this can be cleaned up -- what exactly are you trying to do?